.cspell.json

```diff
@@ -76,6 +76,9 @@
         "hexlify",
         "ctypes",
         "ioapic",
-        "OEMID"
+        "OEMID",
+        "iommu",
+        "IOMMUEFR",
+        "IVMD"
```

Comment:
IOMMU is generic, but are IOMMUEFR and IVMD AMD-specific acronyms?  If yes, then perhaps they should be local to the ivrs_parser*.py files not global in .cspell.json?

Reply:
So I have two choices if I want to keep them inside the ivrs_parser. I could change the name, or I could put comments that disable the spell checker. There's only one word file list. So if we want them to be valid words, this is the only spot they can go.

Reply:
Since there are only 2 AMD-IOMMU-specific files, I'd lean toward the per-file syntax:
# spell-checker:ignore IVMD, IOMMUEFR

But just my opinion 😁

Reply:
Oh that's an option? Let's do that.

Reply:
An excellent suggestion